src/_igraph/graphobject.c

```diff
@@ -13411,7 +13411,7 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "Generates a tree in which almost all vertices have the same number of children.\n\n"
    "@param n: the number of vertices in the graph\n"
    "@param children: the number of children of a vertex in the graph\n"
-   "@param type: determines whether the tree should be directed, and if\n"
+   "@param mode: determines whether the tree should be directed, and if\n"
```

Comment:
Does this change also belong to this PR?

Reply:
Ah, no, sorry that was part of my previous PR that you already merged.